pkg/tunneler/tunnel.go

```diff
@@ -32,12 +32,14 @@ import (
 	"k8s.io/klog/v2"
 
 	workloadv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/workload/v1alpha1"
+	"github.com/kcp-dev/logicalcluster/v3"
 )
 
 const (
-	defaultTunnelPathPrefix = "/services/syncer-tunnels/clusters"
+	// DefaultTunnelPathPrefix is the default path prefix for the tunnel service
+	DefaultTunnelPathPrefix = "/syncer-tunnels"
```

Comment:
why was this moved outside of services? It was intentionally under `/services`.

Reply:
@sttts was there a **strong** reason / hard requirement that made you use the `/services` URL ?

Reply:
`/services` is for extensions. We will certainly not open up `/clusters/<sync target workspace>/<non-resource-url>`, i.e. add non-standard paths in the normal kube apiserver space.

Reply:
We could maybe talk about a subresource of the synctarget.

Reply:
Just a path change you mean ?
And we also have a filter on the the synctarget sub-resource, just like the one we have for the pod sub-resources ?

Reply:
Done, now "connect" and "proxy" are subresources of the synctarget

Reply:
@sttts Is the implementation of this change (as done in commits 69ce74eee7b52fb4df79855e145fc101c32ac632 and 1c43ab5bcff9cd292346ba511a3cdb5247afffe0) OK for you ?